integration-tests/pom.xml

```diff
@@ -87,9 +113,55 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <!-- Not needed for maven build but useful for testing from intellij -->
+          <systemPropertyVariables>
```

Comment:
Couldn't find an easy way to not duplicate this.  Putting this on surefire plugin enables IntelliJ to automatically pick it up.